vnext/ReactUWP/Views/ViewControl.h

```diff
@@ -4,101 +4,35 @@
 #pragma once
 
 #include <Views/ViewPanel.h>
-#include <Views/ShadowNodeBase.h>
 
-#include <winrt/Windows.Foundation.h>
-#include <winrt/Windows.UI.Xaml.h>
 #include <winrt/Windows.UI.Xaml.Automation.h>
 #include <winrt/Windows.UI.Xaml.Automation.Peers.h>
-#include <winrt/Windows.UI.Xaml.Automation.Provider.h>
-#include <winrt/Windows.UI.Xaml.Controls.h>
 
-namespace react {
-namespace uwp {
+#include "cppwinrt/ViewControl.g.h"
+namespace winrt::react::uwp::implementation
+{
 
 //
 // ViewControl is ViewViewManager's ContentControl but with a custom AutomationPeer
 // using DynamicAutomationPeer (See below)
 //
-struct ViewControl : winrt::Windows::UI::Xaml::Controls::ContentControlT<ViewControl>
+struct ViewControl : ViewControlT<ViewControl>
 {
-  using Super = winrt::Windows::UI::Xaml::Controls::ContentControlT<ViewControl>;
-private:
+  using Super = ViewControlT<ViewControl>;
+public:
   // Constructors
   ViewControl();
```

Comment:
does the ctor need to be public or just need a different friend helper?  We had run into some accidental memory leaks before of constructing things with operator new instead of the make helper - just want to be confident we can spot those.

Reply:
I'm trying to move us closer in alignment with a "standard" cppwinrt. If we're going to write in cppwinrt, then we need to get used to the paradigm. However, I'm still trying to determine why we I keep hitting com exceptions when using the `auto control = winrt::react::uwp::ViewControl()`.

Reply:
I've figure it out - our Frankenstein cppwinrt implementation and dll defs mean our ActivateFactory methods are out of whack at the moment.